### ISSUE TITLE
[Bug] [SuperUrlBar] Latest Zen Update broke Blur for non-internal pages

### ISSUE THREAD

**Issue by Swivelgames** [JLBlk/Zen-Themes#65]
**Affected Theme**
 - **Name**: `SuperUrlBar`
 - **Version**: v1.4.2

**Affected Theme Settings**
- `Enable a Background Blur when the Url Bar is focused & select its Intensity`

**Zen Version**: `1.0.2-b.0 (Firefox 133.0)`

**System OS**: Both macOS and Linux 

**Describe the bug**
When the URL Bar is focused and active, blur does not apply to non-internal pages anymore.

**To Reproduce**
1. Install and Enable SuperUrlBar
2. Enable `Enable a Background Blur when the Url Bar is focused & select its Intensity`
3. While in Zen Settings, activate the URL bar and start typing. Notice that the background is correctly blurred
5. Open a new tab and go to Google.com

**Expected behavior**
Background is correctly blurred like it is on the browser-internal page.

![image](https://github.com/user-attachments/assets/44ad06cf-a41e-481d-a2d4-d349a080942d)

**Screenshots/Videos**
Background is not blurred.

![image](https://github.com/user-attachments/assets/8b06626e-b0c5-4539-a109-8b1853d65a5c)

**Additional context**
This was working on a previous version. I noticed it break on macOS first last week, and now on my Linux machine today.

**_It also appears to be off-center, but I didn't check to see if there was an existing bug for that yet._**

**Comment by Swivelgames** [comment]
I hadn't realized it, but I had the now-deprecated official Floating Url Bar extension installed and enabled at the same time. That was the issue :grimacing:

**Comment by Swivelgames** [comment]
Just kidding. It seems to be intermittent... It's very odd.

**Comment by JLBlk** [comment]
Yeah, it's a long onging issue that's related to "clip-path" (used to create the rounded browser corners). The problem with clip-path is that it uses some performance (should be unnoticable though) so they tried to find ways to create the corners without clip-path and suceeded, but now it seems they're kinda using sth like that again so blur isn't working outside browser internal pages.
If you know how to edit your userChrome.css paste the following code to fix it:

```
  /* Scuffed af solution to make blur work (need to find better solution) */
  :root:not([inDOMFullscreen='true']):not([chromehidden~='location']):not([chromehidden~='toolbar']) {
    clip-path: content-box;
  }
```

I probably won't update this officially soon since the SuperUrlBar Mod needs an "Overhaul" after the beta update but i'm rather busy at the time and not really motivated, sry
